index.html

```diff
@@ -18,7 +18,6 @@
 </h3>
 <p>
   Tedious is pure Javascript, as are its dependencies.
-  (Well, the source is <a href="http://jashkenas.github.com/coffee-script/">CoffeeScript</a>.)
   So it should run without change on any platform where Node is supported.
```

Comment:
Yes, you're right, we use babel and transpile when publishing a new release, so even users of Node 0.10 are able to use tedious. 👍